src/action.js

```diff
@@ -182,6 +182,9 @@ async function runCodeseeMap(config, excludeLangs) {
   if (config.apiToken) {
     args.push("-a", config.apiToken);
   }
+
+  args.push("-r", `https://github.com/${config.origin}`);
```

Comment:
Why do all the other args need to be wrapped in an `if (config.origin) {` but not this one?

Reply:
The other options may be falsey. `config.origin` is always set to the value of `process.env.GITHUB_REPOSITORY` which is always a truthy string.

Reply:
That's true today, but may not be true forever, right? Regardless, thanks for explaining this to me. I'm satisfied.